packages/contracts/test/framework/dao/dao-factory.ts

```diff
@@ -291,6 +293,18 @@ describe('DAOFactory: ', function () {
     );
   });
 
+  context('ERC-165', async () => {
```

Comment:
I would also test that `protocolVersion()` returns `1.3.0` here

Reply:
Added